apps/src/lib/genesis.rs

```diff
@@ -12,6 +13,7 @@ use sha2::{Digest, Sha256};
 #[derive(Debug)]
```

Comment:
```rust 
#[derive(Debug)]
pub struct Genesis {
    #[cfg(not(feature="dev"))]
    pub validators: Vec<Validator>,
    #[cfg(feature="dev")]
    pub validators: Validator,
    pub parameters: Parameters,
}
```
is terser than duplicating the struct twice (which also looks more surprising at first)

Reply:
that's is better, applied, thanks!